### DISCUSSION TITLE
Is this repo still being developed?

### DISCUSSION THREAD

**Discussion by yansonghu-lele** [MrNeRF/LichtFeld-Studio#57] [General]
I want to use this repo as the foundation of my research project, but I'm worried that this repo is no longer in development. The last commit was months ago. Will this repo continue to get updates and support?

**Comment by MrNeRF** [comment]
Hey, 
currently I don't have time to work on this code. I can try to answer some questions. As far as I know the code works well, doesn't crash or so. It basically does what it should do. But yes, I am not actively working on this code.

**Reply by yansonghu-lele** [reply]
Thanks for the reply. There isn't many c++ 3D-GS implementations, so I'm happy this exists. There's also OpenSplat, but most research still uses the original GS rasterizer over the gsplat rewrite. I'll try using this and ask questions when needed.

**Reply by MrNeRF** [reply]
Great. I am happy to answer. But keep in mind that it is much more tedious to make progress with C++ than with a python code base.
